.github/workflows/ci.yml

```
@@ -27,10 +37,15 @@ jobs:
       tox-source: "tox"
 
   package:
-    name: Python Package
-    uses: beeware/.github/.github/workflows/python-package-create.yml@main
+    name: Package gbulb
+    permissions:
+      id-token: write
+      contents: read
+      attestations: write
+#    uses: beeware/.github/.github/workflows/python-package-create.yml@main
+    uses: rmartin16/.github-beeware/.github/workflows/python-package-create.yml@hynek-build
```

Looks like this reference still needs to be updated
```suggestion
    uses: beeware/.github/.github/workflows/python-package-create.yml@main
```